examples/usecases/ecommerce-session-based-next-item-prediction-for-fashion.ipynb

```diff
@@ -0,0 +1,1640 @@
+{
```

Comment:
<p>Is this really working as intended without setting <code>seq=True</code> and  <code>max_seq_length=3</code> in the InputBlock? </p><p> </p><p>https://github.com/NVIDIA-Merlin/models/blob/main/merlin/models/tf/inputs/base.py#L47-L48</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/664/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
In the `seq_inputs ` we are setting the parameters seq=True and max_seq_length=3.

In the `cat_inputs `, we do not need them as this is a simple categorical feature, no sequence